examples/hello-world-yaml/test.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: eunomia-hello-world-yaml-demo
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: eunomia-runner-yaml
+  namespace: eunomia-hello-world-yaml-demo
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: eunomia-runner-yaml-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+subjects:
+- kind: ServiceAccount
+  name: eunomia-runner-yaml
+  namespace: eunomia-hello-world-yaml-demo
+---
+apiVersion: eunomia.kohls.io/v1alpha1
+kind: GitOpsConfig
+metadata:
+  name: hello-world-yaml
+spec:
+  templateSource:
+    uri: https://github.com/KohlsTechnology/eunomia
+    ref: master
+    contextDir: examples/hello-world-yaml/template1
+  parameterSource:
+    ref: master
+    contextDir: examples/hello-world-yaml/parameters
+  triggers:
+  - type: Change
+  #- type: Periodic
+    #cron: '*/1 * * * *'
+  serviceAccountRef: eunomia-runner-yaml
+  templateProcessorImage: quay.io/kohlstechnology/eunomia-base:latest
+  resourceHandlingMode: Apply
+  resourceDeletionMode: Delete
```

Comment:
why copy whole contents of already existing files into another file? why not run the same commands as listed in examples/hello-world-yaml/README.md?

Reply:
The whole purpose of this exercise is to use the actual examples. We need to know if any of them stop working due to changes, which will gives us a nice functional coverage on our e2e tests.

Reply:
Ok Got it. Removed file.